source/includes/_lips_booking_events.md

```diff
@@ -0,0 +1,129 @@
+### **BookingEvents**
+<a name="lips_booking_events"></a>
+## Booking Events API
+
+Also referred to as "contact interactions", *events* are activities associated with a contact.  The event is a booking event.
```

Comment:
Ask product to provide something better to say here.

Reply:
Booking events happen when a Contact requests an Appointment through the appointments application.